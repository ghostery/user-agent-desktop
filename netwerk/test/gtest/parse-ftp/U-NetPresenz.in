ftp> open gean5.pfmb.uni-mb.si
Connected to gean5.pfmb.uni-mb.si.
220 NetPresenz v4.1 (Unregistered) awaits your command.
Name (gean5.pfmb.uni-mb.si:cyp): 
331 Guest log in, send Email address (user@host) as password.
230-This file must be in the Startup Messages folder in either the same
    folder as NetPresenz or in the NetPresenz Preferences folder in the
    Preferences folder, and will be displayed to guests user when they log
    in.  Guest users log in by using the username "ftp" or "anonymous".
230 Anonymous login to 1 volumes.  Access restrictions apply.  "/Eng Dept web pages/MariborEnglishClub".
Remote system type is MACOS.
ftp> ls
200 PORT command successful.
150 ASCII transfer started.
-------r--           0     2166     2166 Mar 22 09:21 default.htm
-------r--           0    16504    16504 Mar 21 14:32 FAQs.htm
drwxr-xr-x               folder        5 Mar 22 10:06 FAQs_files
-------r--           0      250      250 Mar 22 09:22 filelist.xml
-------r--           0     1575     1575 Mar 21 14:22 image001.gif
-------r--           0   631827   631827 Mar 21 14:22 image002.jpg
-------r--           0    34318    34318 Mar 21 14:22 image003.jpg
-------r--           0     9121     9121 Mar 22 09:30 MariborEnglishClubHomepage.htm
-------r--           0     6710     6710 Mar 21 14:30 Photos.htm
drwxr-xr-x               folder        5 Mar 22 10:06 Photos_files
-------r--           0     5995     5995 Mar 22 09:23 TOCFrame.htm
-------r--           0     7226     7226 Mar 21 14:27 Upcoming Events.htm
drwxr-xr-x               folder        3 Mar 22 10:06 Upcoming Events_files
226 Transfer complete.
ftp>ftp> close
221 Nice chatting with you.
ftp> 