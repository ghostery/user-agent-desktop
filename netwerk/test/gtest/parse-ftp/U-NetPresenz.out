<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

03-22-2002  09:21:00                 2166 default.htm
03-21-2002  14:32:00                16504 FAQs.htm
03-22-2002  10:06:00       <DIR>          FAQs_files
03-22-2002  09:22:00                  250 filelist.xml
03-21-2002  14:22:00                 1575 image001.gif
03-21-2002  14:22:00               631827 image002.jpg
03-21-2002  14:22:00                34318 image003.jpg
03-22-2002  09:30:00                 9121 MariborEnglishClubHomepage.htm
03-21-2002  14:30:00                 6710 Photos.htm
03-22-2002  10:06:00       <DIR>          Photos_files
03-22-2002  09:23:00                 5995 TOCFrame.htm
03-21-2002  14:27:00                 7226 Upcoming Events.htm
03-22-2002  10:06:00       <DIR>          Upcoming Events_files
